app/views/effective/classifieds/index.html.haml

```diff
@@ -1,9 +1,38 @@
+- classifieds = ::Effective::Classified.classifieds(user: current_user).paginate(page: params[:page], per_page: 12)
```

Comment:
I think this is best fit at the controller but since we're using the ::Effective::Crud, is it just about doing something like the below?

```
def index
  @classifieds = ::Effective::Classified.classifieds(user: current_user).paginate(page: params[:page], per_page: 12)
end
```

Just asking because I'm afraid of overriding something important

Reply:
Yea, forsure!

Could you identify the code it is running, and then we can find out best way to override it

Reply:
Yea, this is a pop quiz. what code is it running when we go to the index action? :)

Reply:
Ooh haha sorry

This is the output that I have for pry's show-method

```
def index
  Rails.logger.info 'Processed by Effective::CrudController#index'

  EffectiveResources.authorize!(self, :index, resource_klass)
  @page_title ||= resource_human_plural_name

  self.resources ||= resource_scope.all if resource_scope.respond_to?(:all)
  @datatable = resource_datatable()

  run_callbacks(:resource_render)
end
```

I was able to change it to the following and it renders, is it the correct way of doing it?

```
def index
  EffectiveResources.authorize!(self, :index, ::Effective::Classified)

  @page_title ||= view_context.classifieds_name_label

  @classifieds = ::Effective::Classified
    .classifieds(user: current_user)
    .paginate(page: params[:page], per_page: EffectiveClassifieds.per_page)
end
```